jenkins/internal-ci/generic/motr.groovy

```diff
@@ -0,0 +1,292 @@
+                                parameters: [
+                                    string(name: 'branch', value: "${branch}")
+                                ]
+                                env.S3_BUILD_NUMBER = s3Build.number
+                            }catch (err) {
+                                build_stage = env.STAGE_NAME
+                                error "Failed to Build S3Server"
+                            }
+                        }
+                    }
+                }
+                            
+                stage ("build Hare") {
+                    steps {
+                        script { build_stage = env.STAGE_NAME }
+                        script {
+                            try {
+                                def hareBuild = build job: 'hare', wait: true,
+                                parameters: [
+                                    string(name: 'branch', value: "${branch}")
+                                ]
+                                env.HARE_BUILD_NUMBER = hareBuild.number
+                            }catch (err){
+                                build_stage = env.STAGE_NAME
+                                error "Failed to Build Hare"
+                            }
+                        }
+                    }
+                }
+            }    
+        }
+    
+        stage ('Tag last_successful') {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                sh label: 'Tag last_successful', script: '''pushd $build_upload_dir/
+                    test -d $build_upload_dir/last_successful && rm -f last_successful
+                    ln -s $build_upload_dir/$BUILD_NUMBER last_successful
+                    popd
+                '''
+                sh label: 'Tag last_successful for dep component', script: '''
+                    # Hare Build 
+                    test -L $build_upload_dir_hare/last_successful && rm -f $build_upload_dir_hare/last_successful
+                    ln -s $build_upload_dir_hare/$HARE_BUILD_NUMBER $build_upload_dir_hare/last_successful
+
+                    # S3Server Build
+                    #test -L $build_upload_dir_s3_dev/last_successful && rm -f $build_upload_dir_s3_dev/last_successful
+                    #ln -s $build_upload_dir_s3_dev/$S3_BUILD_NUMBER $build_upload_dir_s3_dev/last_successful
+                '''
+            }
+        }
+
+        stage ("Release") {
+            when { triggeredBy 'SCMTrigger' }
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                script {
+                    def releaseBuild = build job: 'Release', propagate: true
+                    env.release_build = releaseBuild.number
+                    env.release_build_location="http://cortx-storage.colo.seagate.com/releases/cortx/github/$branch/$os_version/${env.release_build}"
+                    env.cortx_all_image = releaseBuild.buildVariables.cortx_all_image
```

Comment:
is `cortx_all_image` var is suitable name for motr build?

Reply:
We do not have component specific images hence all components are using cortx-all image as of now.